controllers/nodefeaturediscovery_status.go

```diff
@@ -0,0 +1,551 @@
+package controllers
```

Comment:
Ugh, license header is missing from the file

Reply:
ups